core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/perform/ModuleStorageUpdateExecutor.java

```diff
@@ -71,7 +71,7 @@ private void updateModuleConfiguration(String moduleKey, GlobalProperties global
     for (String qpKey : moduleConfiguration.getQprofilePerLanguageMap().values()) {
       if (!qProfileKeys.contains(qpKey)) {
         throw new IllegalStateException(
-          "Module '" + moduleKey + "' is associated to quality profile '" + qpKey + "' that is not in storage. Global storage is probably outdated. Please update binding.");
+          "Module '" + moduleKey + "' is associated to quality profile '" + qpKey + "' that is not in the storage. The SonarQube server binding is probably outdated,  please update it.");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 187 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)